subgraphs/aave-forks/protocols/aave-amm/src/mapping.ts

```diff
@@ -443,8 +439,122 @@ function getAssetPriceInUSDC(
   }
 
   // last resort, should not be touched
-  const inputToken = getOrCreateToken(tokenAddress);
+  const inputToken = Token.load(tokenAddress)!;
```

Comment:
Even though this should never be reached I want to be stricter about leaving potential unwanted crashes, so we should handle the null exception

Reply:
this is part of the code I didn't touch in the refactoring, but what you suggest makes sense.